Tools/XmlDocGen/Tree/Container.cs

```diff
@@ -120,25 +177,41 @@ private void NodeParser(Models.INode node, string path)
             switch (child.Type)
```

Comment:
Can you try to use switch expression instead of switch case construction?

Reply:
I cannot use the switch expression there because of its restrictions, but I improved the readability of switch statements.